tests/cases/dom/manipulate/createElement.js

```diff
@@ -0,0 +1,157 @@
+import QUnit from "qunitjs";
+import {createElement} from "../../../../dom/manipulate";
+
+QUnit.module("dom/manipulate/createElement()",
+    {
+        beforeEach: () =>
+        {
+            document.getElementById("qunit-fixture").innerHTML = "";
+        },
+    }
+);
+
+
+QUnit.test(
+    "with one parameter",
+    (assert) =>
+    {
+        const tagName = "div";
+        const className = "identifier";
+        const element = createElement(tagName);
+        element.classList.add(className);
+        document.getElementById("qunit-fixture").appendChild(element);
+
+        const result = document.getElementsByClassName(className);
+        assert.equal(result.length, 1, "element was only created once");
+        assert.equal(result[0], element, `created <${tagName}>`);
```

Comment:
These tests don't actually test the function, but some integration tests with the DOM.

A better test would be to test the result with `instanceof` (or the type) and that the properties are correctly set.